src/ui/server/constants/routes/insurance/exporter-business.ts

```diff
@@ -4,6 +4,7 @@ const NATURE_OF_BUSINESS_ROOT = '/your-business/nature-of-business';
 export const EXPORTER_BUSINESS = {
   COMPANY_DETAILS_ROOT,
   COMPANY_DETAILS: COMPANY_DETAILS_ROOT,
+  SAVE_AND_BACK_URL: `${COMPANY_DETAILS_ROOT}/save-and-back`,
```

Comment:
I would have as this `COMPANY_DETAILS_SAVE_AND_BACK`

because we'll need other save and back routes for exporter business